harness/determined/common/api/authentication.py

```diff
@@ -38,6 +26,25 @@ def __init__(self, username: str, token: str):
         self.token = token
 
 
+def login(
+    master_address: str,
+    username: str,
+    password: str,
+    cert: Optional[certs.Cert] = None,
+) -> UsernameTokenPair:
+    """
```

Comment:
nit: while you're in here writing doctsrings, I'd consider it a nice bonus if you wrote them in Google style (omitting components as you find them redundant).

Reply:
done